src/components/ui/Ticker.tsx

```diff
@@ -1,10 +1,10 @@
 import React, { useState, useEffect } from 'react';
 import {
-  DollarSign,
-  Pen as Yen,
-  Euro as EuroSign,
-  PoundSterling,
-  Loader,
+  DollarSign as DollarSignIcon,
+  Pen as YenIcon,
+  Euro as EuroSignIcon,
+  PoundSterling as PoundSterlingIcon,
+  Loader as LoaderIcon,
```

Comment:
why are we asserting this icons here? we can just do:
```ts
import {
  DollarSignIcon,
  JapaneseYenIcon, // why is "Pen" used instead of "Japanese Yen"?
  EuroIcon,
  PoundSterlingIcon,
  LoaderIcon,
}
```